packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -325,4 +326,30 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
       throw detailedError;
     }
   }
+
+  /**
+   * Handles the device type specific logic
+   */
+  private async handleDeviceType(deviceType: string, dn: string): Promise<void> {
+    switch (deviceType) {
+      case LoginOption.BROWSER:
+        await this.webCallingService.registerWebCallingLine();
+        this.agentConfig.deviceType = LoginOption.BROWSER;
```

Comment:
This is a common line which can also be put as this.agentConfig.deviceType = deviceType. We don't need to repeat this in every line and also extension case and dn case can be combined

Reply:
My bad, had forgotten to remove it. Will update.